packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -194,11 +199,12 @@ export async function upload({
       ? CONTENT_TYPE_MAP[extension.toLowerCase()]
       : CONTENT_TYPE_MAP.txt
   }
-  const config: any = {
+  const config: PutObjectCommandInput = {
     // windows file paths need to be converted to forward slashes for s3
+    Bucket: sanitizeBucket(bucketName),
     Key: sanitizeKey(filename),
-    Body: fileBytes,
-    ContentType: contentType,
+    Body: fileBytes as stream.Readable | Buffer,
+    ContentType: contentType!,
```

Comment:
nit: is this ! required? how come TypeScript isn't convinced this exists?

Reply:
didn't need this!